articles/root-cause-analysis.md

```diff
@@ -0,0 +1,79 @@
+---
+authorName: Matthias Meid
+authorGithubUsername: matthias-meid
+issue: 98
+title: You Always Have Time for a Proper Root Cause Analysis
+---
+# {{page.title}}
+
+A production issue will pop up, even in solid software. [Don't panic.](https://en.wikipedia.org/wiki/Phrases_from_The_Hitchhiker%27s_Guide_to_the_Galaxy#Don't_Panic)
```

Comment:
Links haben es schwer in einem gedruckten Buch. Du hast zwei Optionen:
1. erstelle eine Datei root-cause-analysis.resources.md und erwähne die Links da drin (im gedruckten Buch wird auf eine Webseite mit den Links verwiesen, in elektronischer Form sind die Links dann einfach am Schluss des Artikels)
2. lösche die beiden Links

Nach meiner Meinung benötigen beide Referenzen keine Links.